core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -134,6 +137,16 @@ oppia.directive('explorationSummaryTile', [function() {
           }
           return result;
         };
+
+        $scope.isWindowLarge = (
```

Comment:
Need to handle the case when mobileCutoffPx is not specified (if so, set it to 0 explicitly).